helm-chart/renku-core/templates/deployment-nginx.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "renku-core.fullname" $ }}-nginx
+  labels:
+    app.kubernetes.io/name: {{ include "renku-core.fullname" $ }}-nginx
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "renku-core.fullname" $ }}-nginx
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "renku-core.fullname" $ }}-nginx
+    spec:
+      containers:
+        - name: nginx
```

Comment:
Please add a [security context](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#securitycontext-v1-core) here and give nginx only what's absolutely needed. Ideally this will at least include 
`allowPrivilegeEscalation = false`
`capabilites = "drop all"`
`runAsNonRoot = false`
I _think_ this should be possible (after swapping to a different port, see below), but I'm not 100% sure.

Reply:
If you meant `runAsNonRoot = true` (false is the default) then it can't be run with the default nginx image since it runs as root.

Reply:
yes, that's indeed what I meant, sorry. Can you try with https://hub.docker.com/r/bitnami/nginx
Usually all the images published by bitnami run as non-root.

Reply:
It worked well. Some configuration changes were needed since this image uses a different config path.